### PR TITLE
feat: changed the color palette + mutliple fixes

### PR DESCRIPTION
**Pull request by GaelFerrand**

### CODE REVIEW

File: ammo-be/src/routers/VersionRouter/index.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import express from 'express';
+
+const VersionRouter = express.Router();
+
+VersionRouter.get(
```
Comment (chandelieraxel) [inline ammo-be/src/routers/VersionRouter/index.ts:5]: Not sure this feature belong inside a "feat/new-palette" branch 🤣

File: ammo-be/src/routers/VersionRouter/index.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    '/version',
+    (req, res): express.Response => {
+        return res.status(200).json({ version: '0.0.1' });
```
Comment (chandelieraxel) [inline ammo-be/src/routers/VersionRouter/index.ts:8]: Should be in .env

File: ammo-be/src/shared/types/Bullet.ts [hunk 3]
```diff
@@ -1,19 +1,12 @@
...
-        | 'GET'
-        | 'PUT'
-        | 'POST'
-        | 'DELETE'
-        | 'HEAD'
-        | 'CONNECT'
-        | 'OPTIONS'
-        | 'TRACE'
-        | 'PATCH';
+    method: HttpMethod;
```
Comment (chandelieraxel) [inline ammo-be/src/shared/types/Bullet.ts:9]: HTTPMethods ?

File: ammo-fe/src/components/buttons/RecorderButton/index.tsx [hunk 4]
```diff
@@ -78,9 +87,9 @@ const RecorderButton = (): ReactElement => {
             >
                 <Button
                     data-cy="recorder-button"
-                    className={`${classes.insideButton} ${
-                        connected ? classes.clickable : ''
-                    }`}
+                    className={`${classes.insideButton}
```
Comment (Spartakyste) [inline ammo-fe/src/components/buttons/RecorderButton/index.tsx:90]: Could put this into a more readable function ?

File: ammo-fe/src/components/buttons/RecorderButton/index.tsx [hunk 5]
```diff
@@ -78,9 +87,9 @@ const RecorderButton = (): ReactElement => {
             >
                 <Button
                     data-cy="recorder-button"
-                    className={`${classes.insideButton} ${
-                        connected ? classes.clickable : ''
-                    }`}
+                    className={`${classes.insideButton} 
+                        ${connected ? classes.clickable : ''} 
+                        ${recording ? '' : classes.off}`}
```
Comment (chandelieraxel) [inline ammo-fe/src/components/buttons/RecorderButton/index.tsx]: Could put this into a more readable function ?

File: ammo-fe/src/util/NetUtil/__tests__/test.ts [hunk 6]
```diff
@@ -13,4 +13,56 @@ describe('NetUtil', () => {
             expect(res).toEqual(PATH);
         });
     });
+
+    describe('isHttpErrorCode', () => {
+        test('Testing informative response status (lower limit) - should return false', () => {
+            const res = isHttpErrorCode(100);
+            expect(res).toEqual(false);
```
Comment (chandelieraxel) [inline ammo-fe/src/util/NetUtil/__tests__/test.ts]: expect(isHttpErrorCode(100)).toEqual(false); ?

**Pull request merged**